spec/reek/configuration/app_configuration_spec.rb

```diff
@@ -99,11 +102,10 @@
       let(:source_via) { "#{directory}/dummy.rb" }
 
       let(:configuration_as_hash) do
-        {
-          directory => { TooManyStatements: { max_statements: 8 } },
-          :IrresponsibleModule => { enabled: false },
-          :TooManyStatements => { max_statements: 15 }
-        }
+        { described_class::DIRECTORIES_KEY =>
+            { directory => { TooManyStatements: { max_statements: 8 } } },
+          'IrresponsibleModule' => { enabled: false },
+          'TooManyStatements' => { max_statements: 15 } }
```

Comment:
I'm not sure why RuboCop isn't complaining but the formatting here is a little off.

Reply:
Moved around the brackets a bit - better?

Reply:
Yes, better!